prow/docs/pr_status_setup.md

```diff
@@ -0,0 +1,57 @@
+# How to setup PR Status 
+This document helps configure PR Status endpoints. 
+
+## Setup secrets
+PR status is an OAuth App that query pull requests on behalf of the authenticated users.
+Therefore, some secret pieces of information are needed to authorize users for the app. The following
+steps will show you how to setup an oauth app that works with PR Status.
+1. Create your Github Oauth application 
+
+    https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/
+    
+    The callback url should be:
+    
+    `<protocol>://<doman_name>:<port>/github-redirect`
```

Comment:
`s/doman/domain`

Should be the `prow` base URL + `/github-redirect`, right? Maybe lets make that more clear